src/main/java/domain/pages/DirectoryPage.java

```diff
@@ -68,4 +73,21 @@ public char getPartyByRepresentative(String representativeName){
         }
         return allStatesArray;
     }
+
+    public List<WebElement> allRowInfoForSelectedState(String state){
```

Comment:
allROwsFor SelecteState .... what does this return? method name should be in line with the fiunction and return of the method

Reply:
this returns a all  row info for the selected state ex: 

1st | Byrne, Bradley | R | 119 CHOB | (202) 225-4931 | Armed ServicesEducation and the WorkforceRules
-- | -- | -- | -- | -- | --


Ardindan bu fonsiyonun dondurdukleri arasindan regex yaparak formata uygun olanlari donduruyorum

Reply:
rename public List String getAllRowsForSelectedState

our methods shoudl NOT return WebElemnet....  this is the layer we are getting rid of webelement layer (domain-dsl etc.)

Reply:
method name changed into getDistrictNamePartyOfficePhoneCommitteeInfoForSelectedState
String yaparken soyle yaptim:
public List<String> getDistrictNamePartyOfficePhoneCommitteeInfoForSelectedState(String state){
        List<String> rows = Arrays.asList(driver.findElements(By.xpath(String.format(rowInfoXpath, state))).toString());
        return rows;
Please share your insights